src/main/java/de/pi3g/pi/oled/OLEDDisplay.java

```diff
@@ -121,43 +134,75 @@
 
     private final I2CBus bus;
     private final I2CDevice device;
+    private final Rotation rotation;
 
     private final byte[] imageBuffer = new byte[(DISPLAY_WIDTH * DISPLAY_HEIGHT) / 8];
 
+    /**
+     * @deprecated Use {@link OLEDDisplay#OLEDDisplay(int, int, Rotation)} instead

```

Comment:
Why is this deprecated? I think it's perfectly alright if someone wants to instantiate the class with default rotation to use this constructor, right?

Reply:
I generally think that heaving a lot of constructors isn't that helpfull and leads to more confusion than it saves time. Thats why I always try to keep the number down. 

My idea here was that most people will just have the display on default bus and adress and use the default constructor or add a rotation if they want to change the orientation. 

The ones that need more configuration will nevertheless have to read some documentation to figure out what adress and and bus number are and how to use them so it won't hurt them to pass an additional arg with the rotation. 

But for backwards compatibillity I still wanted to keep the old constructors for now to keep a smooth transition and remove them at a later stage.

I can totally also see your point for keeping it, so if you prefer that I'll change it.